samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
```

Comment:
Because this sample is down one more directory level than the others in samples/boards, please also make a change to get rid of the message saying this document isn't included in any toctree.  In samples/boards/boards.rst, change the line that says:

     */*

to

     **/*